src/Compilers/Core/Portable/CodeGen/ILBuilderEmit.cs

```diff
@@ -38,13 +38,13 @@ internal void EmitOpCode(ILOpCode code, int stackAdjustment)
 
         internal void EmitToken(string value)
         {
-            uint token = module == null ? 0xFFFF : module.GetFakeStringTokenForIL(value);
+            uint token = module?.GetFakeStringTokenForIL(value) ?? 0xFFFF;
             this.GetCurrentStream().WriteUint(token);
         }
 
         internal void EmitToken(Microsoft.Cci.IReference value, SyntaxNode syntaxNode, DiagnosticBag diagnostics)
         {
-            uint token = module == null ? 0xFFFF : module.GetFakeSymbolTokenForIL(value, syntaxNode, diagnostics);
+            uint token = module?.GetFakeSymbolTokenForIL(value, syntaxNode, diagnostics) ?? 0xFFFF;
```

Comment:
It becomes more readable when you see it more. :-).
The good part is that the default value is the last one, not buried inside the expression. I tend to read this as "and if everything failed, then 0xFFFF"